code/code/spec/spec_objs_deikhan_sword.cc

```diff
@@ -76,11 +76,11 @@ void doBlind(TBeing *ch, TBeing *vict, TObj *o) {
 
 void doSanc(TBeing *ch, TObj *o) {
   affectedData aff;
-  int level=30;
+  int level=50;
 
   aff.type = SPELL_SANCTUARY;
   aff.level = level;
-  aff.duration = Pulse::UPDATES_PER_MUDHOUR/4;
+  aff.duration = Pulse::ONE_SECOND * 60 * 8;
```

Comment:
Is this different than the 3 * Pulse::UPDATES_PER_MUDHOUR for sanc?

Reply:
It's amateur hour over here when I try to fix code. That's probably the normal way and surely much easier to figure. I just grabbed from the list of timers the code has listed and calc'd out real time to mud time the hard way. Backwards now, I realize.

Reply:
Pulse::UPDATES_PER_MUDHOUR * 3

Is that how you'd do it? Still not quite sure how you'd say 3 mud hours.

Reply:
`aff.duration = 3 * Pulse::UPDATES_PER_MUDHOUR;`
is what the sanc code for mortals works out to.